tslint.json

```diff
@@ -1,3 +1,8 @@
 {
-  "extends": "tslint-microsoft-contrib"
+  "extends": "tslint-microsoft-contrib",
+  "rules": {
+	  "no-suspicious-comment": false,
+	  "no-reserved-keywords": false,
+	  "no-relative-imports": false
```

Comment:
We should fix the issue instead of suppressing this rule. WDYT?

Reply:
Your means that we fix all lint issue?

Reply:
@romandev Okay, I remove this ignore rules. :(

Reply:
I don't understand your previous comments exactly :)
I wanted to fix all lint errors in the future (not in this patch).
If you thought that some rules are unnecessary, we can discuss about it together! (in Slack or here)

Reply:
Okay, once the ignore rule has been removed,
After that, it would be better to check if there are any unnecessary rules.